.github/workflows/user_function_deploy.yaml

```diff
@@ -11,10 +11,12 @@ on:
 jobs:
   deploy:
     name: Deploy
-    uses: pagopa/dx/.github/workflows/web_app_deploy.yaml@add-web-app-deploy-workflow
+    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@main
```

Comment:
why this change? cc @lucacavallaro

Reply:
the new workflow is not working properly (maybe I'm missing something)